docs/apache-airflow/concepts.rst

```diff
@@ -1,1708 +0,0 @@
-  of whether there are any retry attempts remaining.
-
-This example illustrates some possibilities
-
-.. code-block:: python
-
-  from airflow.exceptions import AirflowFailException, AirflowSkipException
-
-  def fetch_data():
-      try:
-          data = get_some_data(get_api_key())
-          if not data:
-              # Set state to skipped and do not retry
-              # Downstream task behavior will be determined by trigger rules
-              raise AirflowSkipException("No data available.")
-      except Unauthorized:
-          # If we retry, our api key will still be bad, so don't waste time retrying!
-          # Set state to failed and move on
-          raise AirflowFailException("Our api key is bad!")
-      except TransientError:
-          print("Looks like there was a blip.")
-          # Raise the exception and let the task retry unless max attempts were reached
-          raise
-      handle(data)
-
-  task = PythonOperator(task_id="fetch_data", python_callable=fetch_data, retries=10)
-
-.. seealso::
-    - :ref:`List of Airflow exceptions <pythonapi:exceptions>`
-
-
-Packaged DAGs
-=============
-
-While often you will specify DAGs in a single ``.py`` file it might sometimes
-be required to combine a DAG and its dependencies. For example, you might want
-to combine several DAGs together to version them together or you might want
-to manage them together or you might need an extra module that is not available
-by default on the system you are running Airflow on. To allow this you can create
-a zip file that contains the DAG(s) in the root of the zip file and have the extra
-modules unpacked in directories.
-
-For instance you can create a zip file that looks like this:
-
-.. code-block:: bash
-
-    my_dag1.py
-    my_dag2.py
-    package1/__init__.py
-    package1/functions.py
-
-Airflow will scan the zip file and try to load ``my_dag1.py`` and ``my_dag2.py``.
-It will not go into subdirectories as these are considered to be potential
-packages.
-
-In case you would like to add module dependencies to your DAG you basically would
-do the same, but then it is more suitable to use a virtualenv and pip.
-
-.. code-block:: bash
-
-    virtualenv zip_dag
-    source zip_dag/bin/activate
-
-    mkdir zip_dag_contents
-    cd zip_dag_contents
-
-    pip install --install-option="--install-lib=$PWD" my_useful_package
-    cp ~/my_dag.py .
-
-    zip -r zip_dag.zip *
-
-.. note:: the zip file will be inserted at the beginning of module search list
-   (sys.path) and as such it will be available to any other code that resides
-   within the same interpreter.
-
-.. note:: packaged dags cannot be used with pickling turned on.
-
-.. note:: packaged dags cannot contain dynamic libraries (eg. libz.so) these need
-   to be available on the system if a module needs those. In other words only
-   pure Python modules can be packaged.
-
-
-``.airflowignore``
```

Comment:
I can't find this section right now. Is it expected?

Reply:
It's now at the end of DAGs -> Loading DAGs